src/Agent.Worker/TaskRunner.cs

```diff
@@ -525,5 +542,25 @@ private void PrintTaskMetaData(Definition taskDefinition)
             ExecutionContext.Output($"Help         : {taskDefinition.Data.HelpUrl ?? taskDefinition.Data.HelpMarkDown}");
             ExecutionContext.Output("==============================================================================");
         }
+
+        private bool ContainsSecret(string inputValue)
+        {
+            string maskedString = HostContext.SecretMasker.MaskSecrets(inputValue);
+            return maskedString != inputValue;
+        }
```

Comment:
Could you please add some comments to this method? Also, maybe it makes sense to implement the `IsContain` method for the `SecretMasker` class?